app/models/following.rb

```diff
@@ -0,0 +1,4 @@
+class Following < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :follow, class_name: :User
```

Comment:
`:follower` makes more sense than `:follow`. You generally associate using nouns rather than verbs.